src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/BoundNodeClassWriter.cs

```diff
@@ -894,48 +894,11 @@ private void WriteType(TreeType node)
             {
                 WriteAccept(node.Name);
                 WriteUpdateMethod(node as Node);
-                WriteShallowCloneMethodIfNeeded(node as Node);
             }
 
             WriteClassFooter(node);
         }
 
-        private void WriteShallowCloneMethodIfNeeded(Node node)
```

Comment:
Did this change need to be made as part of this?

Reply:
No, but it sure simplified things, since I was changing the attributes.

---
In reply to: [301353131](https://github.com/dotnet/roslyn/pull/37052#discussion_r301353131) [](ancestors = 301353131)